packages/sanity/schemas/variant.ts

```diff
@@ -108,13 +131,13 @@ export default {
           to: [{ type: "style" }],
         },
       ],
-    },
+    }),
   ],
   preview: {
     select: {
       title: "name",
-      media: "images.0.asset",
+      media: "images.0.images",
```

Comment:
due to the `productImage` changes, this should be accessing the correct image on `asset` rather than `images`
```suggestion
      media: "images.0.asset",
```